concreate/builders/osbs.py

```diff
@@ -206,7 +206,8 @@ def commit(self):
             subprocess.call(["git", "show"])
 
         if not (self.noninteractive or decision("Are you ok with the changes?")):
-            subprocess.call(["bash"])
+            ps1 = os.getenv("PS1", "$ ")
+            subprocess.call(["bash"], env={'PS1': "concreate-%s" % ps1})
```

Comment:
This is a nice idea!

Additionally we should think about writing some "into" text.